compose/Windows.md

```diff
@@ -0,0 +1,91 @@
+# Team Edition Server Installation on Windows
+
+## Preparing your Windows environment
+
+When running on Windows, you need to ensure that WSL is enabled. We also recommend using `podman` with `podman-compose`.
+
+> Note:
+If you run Windows on a virtual machine, please ensure that your host machine supports nested virtualization.
+
+> Hint:
+If using Azure VMs, ensure that
+> - Your VM's `Security type` is set to `Standard`
+> - Its size supports nested virtualization. `D2as_v6` is known to work.
+
+Steps to prepare your Windows environment:
+
+1. Ensure your machine has WSL installed by executing `wsl.exe --status` in PowerShell or Command Prompt. If WSL is not installed,
+   follow the instructions that this command produces.
+2. Ensure the dependencies are installed. We'll need `Git`, `podman`, and `podman-compose`.
+   In turn, `podman-compose` requires Python 3.9 or later, and `pip`. Execute the following PowerShell **not as an administrator** to
+   install everything you need:
+
+```powershell
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
+scoop install git podman python
+pip3 install podman-compose
+```
+
+3. Init and start a new podman machine:
+
+```powershell
+podman machine init --rootful dbeaver-team-edition-machine
+podman machine start dbeaver-team-edition-machine
+```
+
+4. Determine the IP address of your podman machine:
+    1. Execute `wsl.exe -d dbeaver-team-edition-machine`
```

Comment:
Too long instruction for IP address determine. Maybe move to hidden comments and in the notes in the end of article.